src/app/components/PairNavbarInfo/index.tsx

```diff
@@ -118,13 +120,30 @@ export const PairNavbarInfo: React.FC<IPairNavbarInfoProps> = ({ pair }) => {
     }
   }, [lastPrice, pair, candles, loading, dayPrice]);
 
+  useEffect(() => {
+    const symbol =
+      pair[0].trading_pairs === pair[1].trading_pairs
+        ? `${pair[0].quote_symbol}/${pair[0].base_symbol}`
```

Comment:
As discussed, Last Traded Price is not populating for some pairs - see comment on Jira for more details